vmm/src/vstate.rs

```diff
@@ -35,21 +35,21 @@ pub enum Error {
     /// vCPU count is not initialized.
     VcpuCountNotInitialized,
     /// Cannot open the VM file descriptor.
-    VmFd(sys_util::Error),
+    VmFd(std::io::Error),
```

Comment:
`std::io::Error` shows up a lot of times, `use std::io::Error as ...` would help